GoogleAppMeasurement.podspec.json

```diff
@@ -7,7 +7,7 @@
         "GoogleUtilities/MethodSwizzler": "~> 7.7",
         "GoogleUtilities/NSData+zlib": "~> 7.7",
         "GoogleUtilities/Network": "~> 7.7",
-        "nanopb": "~> 2.30908.0"
+        "nanopb": "~> 2.30909.0"
```

Comment:
https://github.com/google/GoogleAppMeasurement/pull/27 made it so that the SPM supports `2.30908.0` and `2.30909.0`.

Would we want to do the same here?

Reply:
Yes. Good catch! 

A `git grep` on the CocoaPods Specs repo shows ARCore is also tied to 2.30908.* so this would break compatibility with them.